OpenSim/Sandbox/MatlabScripts/opensimTableToArray.m

```diff
@@ -0,0 +1,35 @@
+function [data_array,labels] = opensimTableToArray(table)
+
+
+
+
+
+import org.opensim.modeling.*
+
+% allocate an array
+nCol = table.getNumColumns;
+nRow = table.getNumRows;
+data = zeros(nRow, nCol);
+labels = {};
+
+for iCol = 0 : nCol - 1
+
+    for iRow = 0 : nRow - 1
+
+        data(iRow+1,iCol+1) = table.getDependentColumnAtIndex(iCol).getElt(iRow,1);
```

Comment:
@jimmyDunne - Did you encounter any issues avoiding off-by-one errors (OBOEs) here because of 0-indexing in C++ and 1-indexing in Matlab? Are the error messages informative if an OBOE is made? Also, it seems like we need an easier way to extract data from a table—if not in the API, then at least in the "Matlab helper functions" we are planning to release.